api/src/main/java/com/epam/pipeline/manager/pipeline/PipelineRunManager.java

```diff
@@ -840,6 +840,20 @@ public PipelineRun updateRunSids(Long runId, List<RunSid> runSids) {
         return pipelineRun;
     }
 
+    /**
+     * Updates run's tags
+     * @param run {@link PipelineRun} for pipeline run which tagsshould be updated
+     * @return
+     */
+    @Transactional(propagation = Propagation.REQUIRED)
+    public PipelineRun updateTags(final PipelineRun run) {
```

Comment:
lets also create api endpoint for `POST /run/{runId}/tag` to be able to update run tags by api

Reply:
So we will need to change signature of the updateTags method, it should get run id and some TagsVO with maps of tags inside